PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/AttachCamera.rbxmx

```diff
@@ -66,8 +66,8 @@ local function CreateAttachCamera()
 
 
 			camera.Focus = CFrame.new(subjectPosition)
-			module.cframe = CFrame.new(camera.Focus.p - (zoom * newLookVector), camera.Focus.p)
-			self.LastCameraTransform = module.cframe
+			camera.CFrame = CFrame.new(camera.Focus.p - (zoom * newLookVector), camera.Focus.p)
+			self.LastCameraTransform = camera.CFrame
```

Comment:
It does. I'll figure out which release it was added and make sure it's present everywhere.